src/elements/Badge/BadgeTheme.ts

```diff
@@ -33,40 +33,11 @@ const baseTheme: Partial<BadgeTheme> = {
       'bottom-0 left-0 -translate-x-2/4 translate-y-2/4 origin-[0%_100%]',
     'bottom-end':
       'bottom-0 right-0 translate-x-2/4 translate-y-2/4 origin-[100%_100%]'
-  }
-};
-
-export const badgeTheme: BadgeTheme = {
-  base: baseTheme.base,
-  disableMargins: baseTheme.disableMargins,
-  badge: baseTheme.badge,
-  position: baseTheme.position,
-  positions: baseTheme.positions,
-  colors: {
-    default: 'bg-white text-black',
-    primary: 'bg-primary text-text-primary',
-    secondary: 'bg-secondary text-text-primary',
-    error: 'bg-error text-text-primary'
-  }
-};
-
-export const legacyBadgeTheme: BadgeTheme = {
-  base: baseTheme.base,
-  disableMargins: baseTheme.disableMargins,
-  badge: [
-    baseTheme.badge,
-    'p-[var(--spacing-xs)] rounded-[var(--badge-border-radius)] text-[var(--font-size-sm)]'
-  ].join(' '),
-  position: baseTheme.position,
-  positions: baseTheme.positions,
+  },
   colors: {
-    default:
-      'bg-[var(--badge-color-background-default)] text-[var(--badge-color-default)]',
-    primary:
-      'bg-[var(--badge-color-background-primary)] text-[var(--badge-color-primary)]',
-    secondary:
-      'bg-[var(--badge-color-background-secondary)] text-[var(--badge-color-secondary)]',
-    error:
-      'bg-[var(--badge-color-background-error)] text-[var(--badge-color-error)]'
+    default: 'bg-background-neutral-raised-base text-content-text-inverse-base',
+    primary: 'bg-background-brand-base text-content-text-inverse-base',
+    secondary: 'bg-background-neutral-raised-3 text-content-text-inverse-base',
+    error: 'bg-background-semantic-error-base text-content-text-inverse-base'
```

Comment:
these tokens don't look right to me. these look like level 3 color tokens and not level 4 theme level tokens

Reply:
Yes, you are totally right. But that's a caused inconsistency of our DS and Rwablocks. In DS we have only two colors for Indicator:
<img width="348" height="263" alt="Screenshot 2025-08-12 at 10 04 41" src="https://github.com/user-attachments/assets/bcc83c8f-f70e-49d1-9388-81bb45eafc29" />

It's a little bit the same what we have fot Tabs variants.

Reply:
I can add these colors as 4 level tokens, but it will be inconsistent with our DS. So, probably, better to rise it with the design team

Reply:
Yes, that's why we had conversation about DS and Rwablocks inconsistency. We are still confusing with all these things (Chip, Badge, Tag, Indicator and so on)

Reply:
This badge could be totally alone like: <Badge content="7" />
<img width="90" height="70" alt="Screenshot 2025-08-15 at 10 46 56" src="https://github.com/user-attachments/assets/40733bf3-8467-4f88-85d2-ef8f8b4c7916" />